internal/controllers/common/resource_definitions/resource_definitions.go

```diff
@@ -286,14 +302,29 @@ func NewCoreContainer(cr *operatorv1beta1.Cryostat, specs *ServiceSpecs, imageTa
 	mounts := []corev1.VolumeMount{
 		{
 			Name:      cr.Name,
-			MountPath: "flightrecordings",
+			MountPath: configPath,
+			SubPath:   "config",
+		},
+		{
+			Name:      cr.Name,
+			MountPath: archivePath,
 			SubPath:   "flightrecordings",
 		},
 		{
 			Name:      cr.Name,
-			MountPath: "templates",
+			MountPath: templatesPath,
 			SubPath:   "templates",
 		},
+		{
+			Name:      cr.Name,
+			MountPath: clientlibPath,
+			SubPath:   "clientlib",
+		},
+		{
+			Name:      cr.Name,
+			MountPath: "truststore",
+			SubPath:   "truststore",
+		},
```

Comment:
Is this one needed for something? So far we've just been mounting individual certs under /truststore as separate mounts.

Reply:
Mounting the certs individually works for ones that are added to the CR, but without this configuration the user can't always do it manually through the web-client. When I tried to upload a certificate through the web-client, I got a `Permission denied: /truststore/mycert.cer` error response from Cryostat, since the directory itself didn't exist because I hadn't added any certs to the CR.

Although, this does mean that for Cryostats deployed by the Operator, there's now another opportunity for cert content de-sync, right? So we either need to also ensure that whatever two-way state syncing we implement can work for certs too, or we need a better way to disable/hide the cert upload feature from the UI if we want to enforce that it's only done through the CR.

Reply:
Ah, good job catching that. I suppose they would be out-of-sync, but they don't interfere with each other do they? The operator's certificates refer to Secrets, where certificates uploaded through Cryostat are files in the PVC. We wouldn't be able to express these in the Cryostat CRD in its current state.

Reply:
That's true, they don't interfere with each other - unless the user tries to manually upload a cert with the same filename as one that they've also configured via CRD.

Reply:
If this happens they'll get an HTTP 409 response with "certificate already exists", so that seems acceptable IMO.